pkg/openscap/openscap_test.go

```diff
@@ -0,0 +1,70 @@
+package openscap
+
+import (
+	"fmt"
+	//docker "github.com/fsouza/go-dockerclient"
+	"testing"
+)
+
+type testScanner struct {
+	defaultScanner
+
+	getRHELDistOriginal bool
```

Comment:
@pweil- Sorry I don't understand this. How will I call the real ```Scan()``` with mock's of the other functions this way? because once I call ```realScanner.oscapChroot(oscapArgs)```?
Anyway I will have to change the package code again to write tests for the private functions and mock the library calls, right?